lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -244,101 +238,152 @@ spec = do
-        let password = "Secure Passphrase" :: Text
-        let createWallet = Json [json| {
-                "name": "1st Wallet",
-                "mnemonic_sentence": #{mnemonics15},
-                "passphrase": #{password}
-                } |]
-        r0 <- request @ApiWallet ctx ("POST", "v2/wallets") Default createWallet
-        verify r0
-            [ expectResponseCode @IO HTTP.status202
-            , expectFieldEqual walletName "1st Wallet"
-            , expectFieldEqual balanceAvailable 0
-            , expectFieldEqual balanceTotal 0
-            ]
-        let wDest = getFromResponse Prelude.id r0
-        let (Right seed) = fromMnemonic @'[15] @"seed" mnemonics15
-        let pwd = Passphrase $ BA.convert $ T.encodeUtf8 password
-        let rootXPrv = generateKeyFromSeed (seed, mempty) pwd
-        let st = mkSeqState @(Jormungandr 'Testnet)
-                (rootXPrv, pwd) defaultAddressPoolGap
-        let (addrD, st') = genChange pwd st
-        addrs <- listAddresses ctx wDest
-        let destination = (addrs !! 1) ^. #id
-        let amt = 100 :: Natural
-        let payload = Json [json|{
-                "payments": [{
-                    "address": #{destination},
-                    "amount": {
-                        "quantity": #{amt},
-                        "unit": "lovelace"
-                    }
-                }],
-                "passphrase": "Secure Passphrase"
-            }|]
-        let (feeMin, _feeMax) = ctx ^. feeEstimator $ TxDescription
-                { nInputs = 1
-                , nOutputs = 1
-                }
-        r1 <- request @(ApiTransaction t) ctx (postTxEp wSrc) Default payload
-        verify r1
+    postExternalTxTest ctx = do
+        let toSend = 1 :: Natural
+        (ExternalTxFixture wSrc wDest fee txWits) <-
+                fixtureExternalTx ctx toSend
+        let baseOk = Base64
+        let encodedSignedTx = encodeTx txWits MsgTypeTransaction baseOk
+        let payload = NonJson . BL.fromStrict . (toRawBytes baseOk)
+        let headers = Headers [ ("Content-Type", "application/octet-stream") ]
+        r <- request
+            @ApiTxId ctx postExternalTxEp headers (payload encodedSignedTx)
+        verify r
             [ expectSuccess
             , expectResponseCode HTTP.status202
             ]
-        r2 <- request @ApiWallet ctx (getWalletEp wDest) Default Empty
-        verify r2
+
+        rb <- request @ApiWallet ctx (getWalletEp wDest) Default Empty
+        verify rb
             [ expectSuccess
-            , expectEventually ctx balanceAvailable amt
+            , expectEventually ctx balanceAvailable toSend
+            ]
+        ra <- request @ApiWallet ctx (getWalletEp wSrc) Default Empty
+        verify ra
+            [ expectEventually ctx balanceAvailable (faucetAmt - fee - toSend)
+            ]
+
+        let baseWrong = Base16
+        let wronglyEncodedTx = encodeTx txWits MsgTypeTransaction baseWrong
```

Comment:
I acknowledge the comment about the time savings (https://github.com/input-output-hk/cardano-wallet/pull/664#discussion_r319909430) it seems that the test takes only ~4s (on my box) so I think it would be worth splitting it in two...

Reply:
split the test into two